.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
```

Comment:
should you also remove the travis.yml?

fwiw, this should run on travis.com under our account since it's on our org. but I'm in support of moving it, since this one is _so_ simple.

Reply:
Sure, I can do that.